qualang_tools/digital_filters/filters.py

```diff
@@ -88,17 +88,31 @@ def calc_filter_taps(
 
     return _check_hardware_limitation(qop_version, feedforward_taps, list(feedback_taps))
 
-
-def exponential_decay(x, a, t):
```

Comment:
Looks good!
One last comment.
We don't want just to remove existing functionality once it was added, so we should keep `exponential_decay(x, a, t)`, and make it `return multi_exponential_decay(x, [a], [t], s=1)`.
Now there are two options:
1. Either we keep it long term (and then we can also add the `s=1` argument.
2. Or, we add a deprecation warning to it, and remove it in the future.

WDYT?

Oh, and also , need to update the changelog.